doc/source/theory/multiphysics/passive_tracer/passive_tracer.rst

```diff
@@ -0,0 +1,10 @@
+Advection-Diffusion
+--------------------
```

Comment:
We don't have any advection-diffusion purely equation. This is what the Passive Tracer is.
I mean if you want to add theory on the solution fo advection diffusion that's chill